src/api/app/views/webui2/webui/request/_delete_request_dialog.html.haml

```diff
@@ -6,11 +6,17 @@
           %h5.modal-title#delete-request-modal-label Create Delete Request
         .modal-body
           %p
-            -# FIXME: get rid of this helper
-            Do you really want to request the deletion of #{project_or_package_link(project: project.name, package: package.name)}?
+            -# FIXME: get rid of the helper project_or_package_link
+            :ruby
+              link = if defined?(package)
```

Comment:
I know you just c&p but having a generic function which you need to check before what you put into it is a little bit ... pointless? :thinking: :smile:

Reply:
Have a look at line 10. It's not c&p:-)

It looks like [project_or_package_link](https://github.com/openSUSE/open-build-service/blob/6092a895bc1e89f76a43c66a726812f6766bdc68/src/api/app/helpers/webui/webui_helper.rb) already handles this. Is the `defined?` check really needed?

Reply:
Calling `package.name` when `package` is not defined will cause an error. This is why I do `defined?(package)`

Reply:
I see. But what's the point of calling project_or_package_link then. At this point we already know if we need to link to the project or package.

Reply:
I need the same output as what `project_or_package_link` produces. It's just to avoid repeating what it does. That helper would need to be refactored or deleted as I wrote in my comment. However, I don't want to do that in this PR.